install.sh

```diff
@@ -6,12 +6,12 @@ function cleanup() {
   echo "Cleaning up image"
   podman stop -i -t 0 blue-build-installer
   sleep 2
-  podman image rm registry.gitlab.com/wunker-bunker/blue-build:latest-installer
+  podman image rm ghcr.io/blue-build/cli:main-installer
```

Comment:
Oh you'll want that to be latest-installer. The problem is we haven't made a tag with the new GitHub workflow so it's not there yet. This tag would be following the main branch builds directly

Reply:
Oh, yeah, saw that there's no `latest-*` on GHCR, so I just assumed main=main branch=latest. I guess latest means latest release, then?